src/v2/cookbook/unit-testing-vue-components.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Unit Testing Vue Components
+type: cookbook
+order: 1.2
+---
+
+## Simple Example
+
+Unit testing is a fundamental part of software developmenet. The underlying idea is tests the smallest units of code, in isolation from others. This makes it easy to refactor, add new features, or track down bugs. Vue's [single-file components](./single-file-components.html), it is straight forward to write unit tests for components in isolation. This lets you develop new features with confidence you are not breaking existing ones, and helps other people to understand what your component does.
```

Comment:
Great opening, a couple of things:
1) s/developmenet/development
2) Second sentence isn't grammatically correct, how about something like: Unit tests evaluate the smallest units of code in isolation, in order to track down bugs, or make it easier to track down bugs or refactor.
3) No comma necessary after link to sfc- so it would be "Vue's single file components make it straightforward..."